charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -33,6 +33,15 @@ spec:
       tolerations:
       - effect: NoExecute
         operator: Exists
+      initContainers:
+      - name: set-iptable-rules
+        image: alpine:3.8
```

Comment:
Don't hard-code this, we have it in our [`images.yaml`](https://github.com/gardener/gardener/blob/master/charts/images.yaml#L139-L141).